src/Stream.php

```diff
@@ -676,6 +712,61 @@ public function toArray(): array
         return $result;
     }
 
+    /**
+     * Writes iterable source to the file line by line.
+     *
+     * Elements of the iterable source must be stringifiable.
+     *
+     * @param resource $file
+     * @param string $separator (optional) inserted between each item. Ex: ', ' for 1, 2, 3, ...
+     * @param string $prefix    (optional) prepended to string
+     * @param string $suffix    (optional) appended to string
+     *
+     * @return void
+     */
+    public function toFile($file, string $separator = "\n", string $prefix = '', string $suffix = ''): void
```

Comment:
Question: Would it be more cross-platform compatible to use `\PHP_EOL` here instead of `\n` as the default separator here.

Also, I think `prefix` and `suffix` are misnamed here. I think what you want to call these are `header` and `footer`. And based on those names, I think a reasonable person would presume that the newline separator will be in between the header and lines, and the lines and the footer.

Reply:
Done.